llvmlite/tests/test_refprune.py

```diff
@@ -440,34 +440,37 @@ def test_fanout_raise_1(self):
         self.assertEqual(stats.fanout_raise, 2)
 
     fanout_raise_2 = r"""
-define i32 @main(i8* %ptr, i1 %cond) {
+define i32 @main(i8* %ptr, i1 %cond, i8** %excinfo) {
 bb_A:
     call void @NRT_incref(i8* %ptr)
     br i1 %cond, label %bb_B, label %bb_C
 bb_B:
     call void @NRT_decref(i8* %ptr)
     ret i32 0
 bb_C:
-    ret i32 1, !ret_is_a_raise !0           ; bad metadata
+    store i8* null, i8** %excinfo, !numba_excetion_output !0      ; bad metadata
+    ret i32 1
 }
 
 !0 = !{i1 true}
 """
 
     def test_fanout_raise_2(self):
         mod, stats = self.check(self.fanout_raise_2)
-        self.assertEqual(stats.fanout_raise, 0)
+        # Change in behaviour: ignore bad metadata
+        self.assertEqual(stats.fanout_raise, 2)
```

Comment:
CI is failing. The change should be reverted back to:

```suggestion
        self.assertEqual(stats.fanout_raise, 0)
```